2024-11-27 11:08:43,897 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:08:46,912 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:08:55,029 - app.main.routes - ERROR - Error en list_containers: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { user: 1, name: 1 }, name: "unique_name_index", background: false, sparse: true }, existing index: { v: 2, unique: true, key: { name: 1 }, name: "unique_name_index", background: false, sparse: true }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { user: 1, name: 1 }, name: "unique_name_index", background: false, sparse: true }, existing index: { v: 2, unique: true, key: { name: 1 }, name: "unique_name_index", background: false, sparse: true }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1732702134, 5), 'signature': {'hash': b'\xb9\x9b\xb0\xb9\xa0\xc1\x9f\xcc\x13#G\xe5\xeeA1\xa9Z\xf0q\xcb', 'keyId': 7377441126493454338}}, 'operationTime': Timestamp(1732702134, 5)}
2024-11-27 11:08:55,029 - app.main.routes - ERROR - Error en list_containers: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { user: 1, name: 1 }, name: "unique_name_index", background: false, sparse: true }, existing index: { v: 2, unique: true, key: { name: 1 }, name: "unique_name_index", background: false, sparse: true }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { user: 1, name: 1 }, name: "unique_name_index", background: false, sparse: true }, existing index: { v: 2, unique: true, key: { name: 1 }, name: "unique_name_index", background: false, sparse: true }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1732702134, 5), 'signature': {'hash': b'\xb9\x9b\xb0\xb9\xa0\xc1\x9f\xcc\x13#G\xe5\xeeA1\xa9Z\xf0q\xcb', 'keyId': 7377441126493454338}}, 'operationTime': Timestamp(1732702134, 5)}
2024-11-27 11:09:21,562 - app.utils - INFO - Imagen guardada exitosamente en: /Users/edefrutos/AWS_Instancia_NO_BORRAR/aws_edfcatalogoqr/app/static/container_pics/cbcb3bb96a1b3105.jpg
2024-11-27 11:09:21,562 - app.utils - INFO - Imagen guardada exitosamente en: /Users/edefrutos/AWS_Instancia_NO_BORRAR/aws_edfcatalogoqr/app/static/container_pics/cbcb3bb96a1b3105.jpg
2024-11-27 11:09:28,439 - app.main.routes - ERROR - Error en print_detail: object of type 'builtin_function_or_method' has no len()
2024-11-27 11:09:28,439 - app.main.routes - ERROR - Error en print_detail: object of type 'builtin_function_or_method' has no len()
2024-11-27 11:11:54,788 - app.main.routes - ERROR - Error en print_detail: object of type 'builtin_function_or_method' has no len()
2024-11-27 11:11:54,788 - app.main.routes - ERROR - Error en print_detail: object of type 'builtin_function_or_method' has no len()
2024-11-27 11:21:20,704 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:22:30,293 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:23:20,961 - app.main.routes - ERROR - Contenedor 6726013bbe87afa8b08bde43 no encontrado
2024-11-27 11:23:20,961 - app.main.routes - ERROR - Contenedor 6726013bbe87afa8b08bde43 no encontrado
2024-11-27 11:43:53,177 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:53:15,814 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:53:30,376 - app - INFO - Aplicación Flask iniciada correctamente
2024-11-27 11:53:33,311 - app - INFO - Aplicación Flask iniciada correctamente
